### PR TITLE
Minor changes to `Ubuntu Docker Scripts for Building IOT-platform-SDK`

### PR DESCRIPTION
**Pull request by DukeDavis12**
1. http typo mistake on line 7.

   2. Removed all trailing whitespaces.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: build/build.sh [hunk 1]
```diff
@@ -4,19 +4,19 @@ export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
...
-export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$http_proxy_port"
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
 
 REMOTE_URL=https://github.com/secure-device-onboard/iot-platform-sdk
 REMOTE_BRANCH=master
 
-if [ "$use_remote" = "1" ]; 
-    then 
+if [ "$use_remote" = "1" ];
+    then
```
Comment (trbehera) [inline build/build.sh:13]: Keep 'then' at the same indent level of if.

if [ condition ]; then
<indent>statement
else
<indent>statement
fi

or 

if [ condition ]; 
then
<indent>statement
else
<indent>statement
fi

File: build/Dockerfile [hunk 2]
```diff
@@ -3,7 +3,7 @@ FROM ubuntu:bionic
 RUN apt update && \
```
Comment (trbehera) [inline build/Dockerfile:3]: Current docker-compose execution shows some warnings while using 'apt', use 'apt-get' instead.

File: build/build.sh (hunk 1 above)
Comment (DukeDavis12) [reply build/build.sh]: Done. Indentation changed.

File: build/build.sh [hunk 3]
```diff
@@ -4,19 +4,20 @@ export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
...
-        git clone $REMOTE_URL
-        cd /tmp/iot-platform-sdk/ 
-        git checkout $REMOTE_BRANCH
-        mvn clean install 
-        cd demo && find . -name \*.war -exec cp --parents {} /home/sdouser/iot-platform-sdk/demo \;      
-    else 
-        mvn clean install 
-fi 
\ No newline at end of file
+if [ "$use_remote" = "1" ]; then
```
Comment (trbehera) [inline build/build.sh:12]: Use 2-space for indenting.

File: build/build.sh [hunk 4]
```diff
@@ -4,19 +4,20 @@ export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
...
+    echo "Building $REMOTE_URL : $REMOTE_BRANCH"
+    cd /tmp/
+    git clone $REMOTE_URL
+    cd /tmp/iot-platform-sdk/
+    git checkout $REMOTE_BRANCH
+    mvn clean install
+    cd demo && find . -name \*.war -exec cp --parents {} /home/sdouser/iot-platform-sdk/demo \;
+ else
```
Comment (trbehera) [inline build/build.sh:20]: There seems to be a space before else, please remove.

File: build/build.sh (hunk 3 above)
Comment (DukeDavis12) [reply build/build.sh:12]: Changed the indentation to 2-space style.

File: build/build.sh (hunk 4 above)
Comment (DukeDavis12) [reply build/build.sh]: Removed the space.

**Pull request merged**